<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-5BSDQGDQ" failures="0" tests="6" name="testNGAnnotation_flags.ActitimeDataProvider" time="8.341" errors="1" timestamp="2023-07-26T15:21:43 IST" skipped="0">
  <testcase classname="testNGAnnotation_flags.ActitimeDataProvider" name="invaildLoginMethod" time="1.435"/>
  <testcase classname="testNGAnnotation_flags.ActitimeDataProvider" name="invaildLoginMethod" time="1.545"/>
  <testcase classname="testNGAnnotation_flags.ActitimeDataProvider" name="invaildLoginMethod" time="1.700"/>
  <testcase classname="testNGAnnotation_flags.ActitimeDataProvider" name="f" time="0.016">
    <error message="
[public void testNGAnnotation_flags.ActitimeDataProvider.f(java.lang.Integer,java.lang.String)] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: f([java.lang.Integer arg0, java.lang.String arg1])
Arguments: [(java.lang.String) ad_main,(java.lang.String) mana_ger]" type="org.testng.internal.reflect.MethodMatcherException">
      <![CDATA[org.testng.internal.reflect.MethodMatcherException: 
[public void testNGAnnotation_flags.ActitimeDataProvider.f(java.lang.Integer,java.lang.String)] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: f([java.lang.Integer arg0, java.lang.String arg1])
Arguments: [(java.lang.String) ad_main,(java.lang.String) mana_ger]
at org.testng.internal.reflect.DataProviderMethodMatcher.getConformingArguments(DataProviderMethodMatcher.java:46)
at org.testng.internal.Parameters.injectParameters(Parameters.java:909)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:37)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- f -->
  <testcase classname="testNGAnnotation_flags.ActitimeDataProvider" name="invaildLoginMethod" time="2.328"/>
  <testcase classname="testNGAnnotation_flags.ActitimeDataProvider" name="invaildLoginMethod" time="1.317"/>
</testsuite> <!-- testNGAnnotation_flags.ActitimeDataProvider -->
